### ISSUE TITLE
torpor multipliers are ignored by the game

### ISSUE THREAD

**Issue by cadon** [cadon/ARKStatsExtractor#942]
ARK seems to ignore stat-multipliers for torpidity. Ignore them in ASB as well.
Probably related to #184.

**Comment by VolatilePulse** [comment]
Ark doesn't ignore the Torpor stat multipliers, Torpor is just calculated differently than normal stats. Unlike other stats, Torpor is not leveled by creatures and it doesn't have a WildLevel value either, it is always zero from what I've seen. This is why the multipliers appear to have no affect.

**Comment by cadon** [comment]
Thanks for the clarification, I was gonna ask you how ARK exactly handles this. So far I assume:

TLDR: IwM should be assumed to be 1, regardless what the user inputs. The other multipliers work like on other stats.

* Base value is the same as for other stats and can be changed with the multiplier
* IwM: the usually used value is 0 and overruled by a value that seems to be 0.06 for all species, hence the multiplier IwM has no effect on the calculation and should be assumed to be 1 (regardless what the value in the setting is).
* IdM: for all known species it's not possible to level torpidity, but it probably works like other stats
* TaM: Ta seems to be 0.5 for all species, TaM works like on other stats
* TmM: Tm seems to be 0 for all species, but probably TmM works like on other stats.

**Comment by VolatilePulse** [comment]
For the way ASB handles Torpor for the moment, yes, IwM for Torpor should always be 1. 

Technically, Torpor's Iw always 0. However, Ark uses an attribute called TheMaxTorporIncreasePerBaseLevel or something along those lines to calculate a creature's final Torpor value. This doesn't follow the traditional stat calculation of other stats as it has its own stat calculation function.

Since all of the stat values for Torpor behave like the other stats, the Torpor's Iw value is replaced with the above mentioned property in the JSON files so ASB can use it like the other stats during calculation.
